### ISSUE TITLE
Email alerts not working

### ISSUE THREAD

**Issue by bakavic** [metabase/metabase#11534]
**Describe the bug**
After email alerts are set in metabase, the usual "You set up an alert" email message does not get sent out.

Also, old email alerts previously set up do not fire.

**Logs**
After setting up an email alert:

```
12-17 02:04:49 ERROR email.messages :: Failed to send message to 'myemail@email.com' with subject 'You set up an alert'
java.lang.IllegalArgumentException: No implementation of method: :render of protocol: #'stencil.ast/ASTNode found for class: nil
	at clojure.core$_cache_protocol_fn.invokeStatic(core_deftype.clj:583)
	at clojure.core$_cache_protocol_fn.invoke(core_deftype.clj:575)
	at stencil.ast$fn__30303$G__30298__30312.invoke(ast.clj:19)
	at stencil.core$render.invokeStatic(core.clj:80)
	at stencil.core$render.invoke(core.clj:74)
	at stencil.core$render_file.invokeStatic(core.clj:87)
	at stencil.core$render_file.invoke(core.clj:83)
	at metabase.email.messages$send_email_BANG_$fn__30768.invoke(messages.clj:340)
	at clojure.core$binding_conveyor_fn$fn__5754.invoke(core.clj:2030)
	at clojure.lang.AFn.call(AFn.java:18)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
```
At the start of an hour:

```
12-17 02:00:00 ERROR  :: SendPulses task failed
java.lang.IllegalStateException: Attempting to call unbound fn: #'metabase.task.send-pulses/monthday
	at clojure.lang.Var$Unbound.throwArity(Var.java:45)
	at clojure.lang.AFn.invoke(AFn.java:32)
	at metabase.task.send_pulses.SendPulses$fn__68023.invoke(send_pulses.clj:98)
	at metabase.models.task_history$fn__41262$do_with_task_history__41267$fn__41268.invoke(task_history.clj:77)
	at metabase.models.task_history$fn__41262$do_with_task_history__41267.invoke(task_history.clj:72)
	at metabase.task.send_pulses.SendPulses.execute(send_pulses.clj:86)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:213)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:557)
```

**To Reproduce**
1. Set up an email alert
2. Check logs, and error message is there.

**Expected behavior**
Email alerts should be sent out.

**Information about your Metabase Installation:**

You can get this information by going to Admin -> Troubleshooting.

- Your browser and the version: 78.0.3904.108
- Your operating system: OS X 10.15.1
- Your databases: MySQL
- Metabase version: v0.33.7.1
- Metabase hosting environment: Docker
- Metabase internal database: MySQL

**Severity**
This is a showstopper for me - I'm going back to the last know working version for me - v0.33.6

**Comment by flamber** [comment]
Hi @bakavic 
Since you don't provide any information, I'm guessing you're on 0.33.7.1, which has a known problem (#11525), that was fixed and a new release 0.33.7.3 was created a few minutes ago, so upgrade to that.
Please close this issue, if you're using 0.33.7.1 - otherwise provide more details.

**Comment by bakavic** [comment]
ah ok, I was searching for "email alerts", guess that why the other Issue didn't come out. Closing.
